config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  devise_for :users
   get 'comments/new'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
   # Defines the root path route ("/")
```

Comment:
- [ ] [OPTIONAL]
Generally, if a user not signed in, the user should be redirected to the login page, however, when enter the app it always assumes that the user is signed in. Let's endeavor to improve upon this aspect.

**Root URL blank:**
![image](https://github.com/emarkees/Gist_Blog/assets/12745474/608aee2b-c9bc-4ce0-82b3-d51b27d7af40)